app/config/config.yml

```diff
@@ -67,3 +66,13 @@ swiftmailer:
     username: '%mailer_user%'
     password: '%mailer_password%'
     spool: { type: memory }
+
+# Redis Configuration
+snc_redis:
+    clients:
+        session:
+            type: predis
+            alias: session
+            dsn: 'redis://%redis_host%'
```

Comment:
How get the value of redis_host? Variable environment?

Reply:
Actually is from the file [parameters.yml](https://github.com/QEdu/qedu-hub/blob/ba62ebae3f4e91a24d77c338c959fe082b1f036a/app/config/parameters.yml.dist#L21), but in the future would be nice have it in environment variables.